### PR TITLE
UA (Ukraine language) translate

### PR DESCRIPTION
**Pull request by Valeriy-Lednikov**
I don't know the structure of the project, but I would like to add the Ukrainian language. Made a translation.

### CODE REVIEW

**Comment by bookmist** [comment]
Move file to the "user" folder. The "system" folder contains only languages, presented in real PS Vita

File: lang/system/ua.xml [hunk 1]
```diff
@@ -0,0 +1,910 @@
...
+			<hide_show_control>Натиснути лівий стік або трикутник</hide_show_control>
+			<exit_manual>закрити довідку</exit_manual>
+			<exit_manual_control>Натиснути Esc або PS</exit_manual_control>
+		</help>
+	</live_area>
+
+	<performance_overlay>
+		<avg>Середній</avg>
+		<min>Мінімум</min>
+		<max>Максимум</max>
```
Comment (Croden1999) [inline lang/system/ua.xml:525]: the text is overrun

File: lang/system/ua.xml [hunk 2]
```diff
@@ -0,0 +1,910 @@
...
+	</settings_dialog>
+
+	<sys_apps_title>
+		<browser>Браузер</browser>
+		<internet_browser>Інтернет-браузер</internet_browser>
+		<trophies>Трофеї</trophies>
+		<trophy_collection>Колекція трофеїв</trophy_collection>
+		<settings>Налаштування</settings>
+		<content_manager>Керування вмістом</content_manager>
+	</sys_apps_title>
```
Comment (Croden1999) [inline lang/system/ua.xml:830]: remove `sys_apps_title`, this language isn't exist in param.sfo

**Review by Croden1999 — CHANGES_REQUESTED** [review]

**Comment by GameRuleUA** [comment]
Valeriy-Lednikov, hello, can you give me link to download this file? I just want to add Ukrainian language into Vita3K

**Comment by Macdu** [comment]
Superseded by #3251, thank you for your contribution!

**Pull request closed without merging**